packages/cli/src/lib/setup/setupPacketManager.ts

```diff
@@ -28,6 +27,13 @@ export class PacketManager {
     private dpkg: boolean;
     private sudo: boolean;
     private readonly _readyPromise: Promise<void>;
+    private readonly COMMANDS = {
+        listUpgradeable: {
+            apt: 'list --upgradeable',
+            'apt-get': 'list --upgradeable',
```

Comment:
But why we need apt-get? 

only I found is "apt-get --simulate upgrade"

Reply:
I am not an expert, maybe all systems which have `apt-get` also have the `apt` cli? Not sure about this, then it would be save to remove `apt-get` entirely. As @AlCalzone had some experience with the package manager, maybe he knows.

Reply:
I also "Think" that both should be there ...

Reply:
Does this help?
https://askubuntu.com/questions/445384/what-is-the-difference-between-apt-and-apt-get/446484#446484

> apt-get may be considered as lower-level and "back-end", and support other APT-based tools. apt is designed for end-users (human) and its output may be changed between versions.

> Note from apt(8):

> 
> The `apt` command is meant to be pleasant for end users and does not need
>        to be backward compatible like apt-get(8).
>

Reply:
https://itsfoss.com/apt-vs-apt-get-difference/
> the first stable version of apt was released in the year 2014, but people started noticing it in 2016 with the release of Ubuntu 16.04

So unless we want to support some really old systems, only using `apt` should be okay I think.